apps/docs/content/components/Avatar/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Avatar
+description: Use Avatar to display a thumbnail representation of a person.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![An image showing two different avatars, a circular and a square one.](https://user-images.githubusercontent.com/175638/234820833-c803cb11-2bc3-4597-88f3-2cf630a46702.png)
+
+## Usage
+
+Use avatars to display a thumbnail that represents individuals in various contexts, such as testimonials, speaker sessions, blog post authors, and beyond.
+
+## Options
+
+### Size
+
+Avatar offers various sizes to fit different layout and visual scale needs. Sizes range from 32 to 80 pixels. Ensure the avatar is proportionate to its accompanying text.
+
+![An image showing the size scale of the avatar component: 32, 40, 48, 64, and 80 pixels](https://user-images.githubusercontent.com/175638/234826575-1d1d5019-f714-41b1-b8bc-c0d312a46cfe.png)
+
+### Shape
+
+Avatars appear as two different shapes: circle and square. The choice depends on the desired visual impression and the overall design aesthetic of your project.
```

Comment:
Just checking that our Brand usage for avatar differs from product?

E.g. this is what the product avatar states:

- Circle avatars represent individual people.
- Square avatars represent non-human entities, such as bots, teams, or organizations

I recall for Galaxy recently, that we used square avatars for people, so assuming this is the case?

Reply:
Yes, it differs from product. We use both circle and square shaped avatars in Galaxy.